app/app.go

```diff
@@ -573,8 +554,8 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 // Regenapp. It is useful for tests and clients who do not want to construct the
 // full Regenapp
 func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
-	config := MakeEncodingConfig()
-	return config.Codec, config.Amino
+	cfg := MakeEncodingConfig()
+	return cfg.Codec, cfg.Amino
 }
```

Comment:
It doesn't look like the bridge service is using this and `MakeEcnodingConfig` is public and the bridge service is using that so this seems unnecessary to keep.

Reply:
👍🏻 removed https://github.com/regen-network/regen-ledger/pull/1357/commits/35a6b5bf66d07acf83090e0d7757933d627d6717